include/mrdox/Config.hpp

```diff
@@ -42,25 +42,59 @@ class MRDOX_DECL
     Config() noexcept;
 
 public:
-    struct Settings
+    struct ExtractOptions
```

Comment:
Should this be public? Would a plugin need to see it? I'm guessing no. Plugins should display what is extracted. I think. So these settings would be an implementation detail and visible to the AST traversal code.